quickwit-cli/tests/cli.rs

```diff
@@ -797,7 +797,7 @@ async fn test_all_local_index() -> Result<()> {
     assert_eq!(result["num_hits"], Value::Number(Number::from(2i64)));
 
     let search_stream_response = reqwest::get(format!(
-        "http://127.0.0.1:{}/api/v1/{}/search/stream?query=level:info&output_format=csv&fast_field=ts",
+        "http://127.0.0.1:{}/api/v1/{}/search/stream?query=level:info&output_format=Csv&fast_field=ts",
```

Comment:
This is the only thing that is a bit annoying here, we need to give `Csv` or `ClickHouseBinaryFormat` for the output format. I'm ok with that.

Reply:
Can't we apply `#[serde(rename_all = snake_case)]`? on the `OutputFormat` enum?

Reply:
Yes, you're right!